uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/LockManagerImpl.java

```diff
@@ -273,21 +273,33 @@ private boolean isLockedByCurrentUser() {
         return lockInfo.isLocked() && lockInfo.lockedBy().equals(user.getIdentifier());
     }
 
-    private void updateLockInfo(@Observes LockInfo lockInfo) {
-        if (lockTarget != null && lockInfo.getFile().equals(lockTarget.getPath())) {
+    void updateLockInfo(final @Observes LockInfo lockInfo) {
+        if (getLockTarget() != null && lockInfo.getFile().toURI().equals(lockTarget.getPath().toURI())) {
```

Comment:
You make a point of using ```toURI()``` on the paths here; but.....

Reply:
@manstis Here we can have a `PathImpl` being compared to an `ObservablePath` (that's what we're avoiding), and......